Source/Core/Core/IOS/Network/KD/NWC24Config.cpp

```diff
@@ -216,4 +216,21 @@ void NWC24Config::SetEmail(const char* email)
   strncpy(m_data.email, email, MAX_EMAIL_LENGTH);
   m_data.email[MAX_EMAIL_LENGTH - 1] = '\0';
 }
+
+std::string NWC24Config::GetAccountURL() const
+{
+  return {m_data.http_urls[0]};
```

Comment:
Please ensure that m_data.http_urls[0] is null-terminated (or use strnlen(str, MAX_URL_LENGTH) to ensure that we only read MAX_URL_LENGTH chars at most).

Reply:
Should we fix the config file if the URL is greater than `MAX_URL_LENGTH`?

Reply:
Sure, why not.